YCloudYUSA/y_lb
# [DS-1498] Add template for preview of the Landing Page with LB

Jira issue: [DS-1498](https://yusa.atlassian.net/browse/DS-1498)

#### Steps to check

- log as admin
- go to edit any Landing Page with LB
- click by "Preview" button
- verify that it looks good

![image](https://github.com/YCloudYUSA/y_lb/assets/744406/cec32a74-0853-4676-9be0-79addf1bb50a)
